bookmark-it/background.js

```diff
@@ -0,0 +1,64 @@
+var currentTab;
+var currentBookmark;
+
+/*
+ * Updates the browserAction icon to reflect whether the current page
+ * is already bookmarked.
+ */
+function updateIcon() {
+  chrome.browserAction.setIcon({
+    path: currentBookmark ? {
+      19: "icons/star-filled-19.png",
+      38: "icons/star-filled-38.png"
+    } : {
+      19: "icons/star-empty-19.png",
+      38: "icons/star-empty-38.png"
+    },
+    tabId: currentTab.id
+  });
+}
+
+/*
+ * Add or remove the bookmark on the current page.
+ */
+function toggleBookmark() {
+  if(currentBookmark) {
```

Comment:
nit: I usually see a space between "if" and "(condition)":

    if (condition)

...there's another one of these at line 44.